siriuspy/siriuspy/devices/psconv.py

```diff
@@ -197,11 +205,18 @@ def _get_devices(devname, proptype):
 
     @staticmethod
     def _get_dev_if_dipole(devname):
-        if devname.dev in {'B', 'B1B2'}:
+        if devname.dis in {'B', 'B1B2'}:
```

Comment:
Isn't that a bug?

Reply:
It is! a very serious one! wondering how this happened... strange. thanks!!!

Reply:
fixed it!